resources/views/backend/news/index.blade.php

```diff
@@ -51,18 +34,24 @@
                   </div>
                 </form> 
               </div>
+              <div class="btn-addnews">
+                <a href="{{ route('news.create') }}" class="btn btn-primary">
+                  <i class="fa fa-plus-circle"></i>
+                  {{__('Add News')}}
+                </a>
+              </div>
             </div>
             <div class="box-body cl">
               @include('flash::message')
               <table class="table table-bordered table-striped clearfix" id="newstable">
                 <thead>
                 <tr>
-                  <th>{{__('Id')}}</th>
-                  <th>{{__('Title')}}</th>
-                  <th>{{__('Content')}}</th>
-                  <th>{{__('Category_id')}}</th>
-                  <th>{{__('Category')}}</th>
-                  <th>{{__('Option')}}</th>
+                  <th class="col-news-id">{{__('Id')}}</th>
+                  <th class="col-news-title">{{__('Title')}}</th>
+                  <th class="col-news-content">{{__('Content')}}</th>
+                  <th class="col-news-categoryid">{{__('Category_id')}}</th>
+                  <th class="col-news-category">{{__('Category')}}</th>
+                  <th class="col-option">{{__('Option')}}</th>
```

Comment:
I think this table will be auto scale, no need add css in this table. Or if you need pls using css of bootstrap like ```col-md-2```